docs/source/release_notes/release-notes.rst

```diff
@@ -52,6 +52,13 @@ develop
          - ``kvs-slow-log`` now uses ``logsafe`` to support sls-compatible logging.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2222>`__)
 
+    *    - |new|
+         - AtlasDB introduces a top-level ``namespace`` configuration parameter, which is used to set the keyspace in Cassandra *and* the client in TimeLock.
+           It is suggested that users update their configuration to use this parameter instead of separate ``keyspace`` and ``client`` configuration parameters if possible; this was done to avoid risks of data corruption if these are accidentally changed independently.
```

Comment:
Yes. It might be worth including a line saying to contact the AtlasDB team if assistance is required (e.g. to perform a timelock client change)